app/main/posts/views/post-view-data.html

```diff
@@ -66,7 +66,7 @@ <h1 class="mode-context-title"><a href="/" ng-bind="nav.site.name"></a></h1>
         <div class="listing timeline init">
 
         <div class="bulk-action" ng-if="posts.length > 0 || !isLoading()">
-            <div class="bulk-action-primary" translate="post.posts_total" translate-values="{'posts': posts.length, 'total_nb': totalItems }"></div>
+            <div class="bulk-action-primary" translate="post.posts_total" translate-values="{'posts': posts.length, 'total_nb': posts.length }"></div>
```

Comment:
ah. This works fine under 20 posts, but once we get over that limit, it will always say x of 20 (instead of *x of realTotal*). Sorry :(

Reply:
Ha, thought that was too easy...